spec/helpers/step_nav_actions_helper_spec.rb

```diff
@@ -72,4 +72,35 @@
       expect(helper.can_submit_2i_review?(step_by_step_page, reviewer_user)).to be true
     end
   end
+
+  describe "#can_revert_to_draft?" do
+    let(:step_by_step_page) { create(:step_by_step_page) }
+
+    it "can reverted to draft if it is approved 2i" do
```

Comment:
```suggestion
    it "can be reverted to draft if it is approved 2i" do
```

Reply:
This has already been done. I didn't change any logic about showing _Revert to draft_ when the status is `approved_2i`. I only moved it into the method, so I didn't think there is a need to explain that here. Maybe I should split it into 2 commits so it's clearer what has been added?